src/App.js

```diff
@@ -1,5 +1,90 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
```

Comment:
이렇게 하면 코드가 더 짧아져서 보기 더 깔끔한 것 같아요!

```suggestion
import { Console, Random } from '@woowacourse/mission-utils';
```